theme/themes/pastanaga/extras/contents.less

```diff
@@ -73,9 +73,28 @@
     padding-top: 4px;
   }
 
+  .ui.table .iconAlignName {
```

Comment:
Although it can be unconsistent in some places, but we are going to - separated CSS classnames everywhere. Can you amend that? Also keep that in mind for next ones.

Reply:
ok